### PR TITLE
Reconstructing the user manual

### PR DESCRIPTION
**Pull request by haoel**
The major work just reconstruct the Probe section by moving the contents from the `configuration` section into the `probe` section

### CODE REVIEW

File: docs/Manual.md [hunk 1]
```diff
@@ -631,53 +1288,25 @@ The following environment variables can be used to fine-tune the request to the
...
-- `timeout` - the maximum time to wait for the probe to complete. default: `30s`.
-- `interval` - the interval time to run the probe. default: `1m`.
-- `failure` - number of consecutive failed probes needed to determine the status down, default: 1
-- `success` - number of consecutive successful probes needed to determine the status up, default: 1
+```yaml
+version: v1.5.0
+```
 
-## 7.1 HTTP Probe Configuration
+The full configuration template you can get from [here](../resources/config.yaml)
```
Comment (proditis) [inline docs/Manual.md:1304]:
```suggestion
You can find the full configuration template [here](../resources/config.yaml)
```

File: docs/Manual.md [hunk 2]
```diff
@@ -631,53 +1288,25 @@ The following environment variables can be used to fine-tune the request to the
...
-```yaml
-version: v1.5.0
+```shell
+easeprobe -f /path/to/conf
 ```
 
-The following example configurations illustrate the EaseProbe supported features.
 
-**Note**:   All probes have the following options:
+And the configuration file should be versioned, the version should be aligned with the EaseProbe binary version.
```
Comment (proditis) [inline docs/Manual.md:1298]:
```suggestion
The configuration file should be versioned, the version should be aligned with the EaseProbe binary version.
```

File: docs/Manual.md [hunk 3]
```diff
@@ -631,53 +1288,25 @@ The following environment variables can be used to fine-tune the request to the
 * `HTTP_AUTHORIZATION`
 * `HTTP_TIMEOUT`
 
-And the configuration file should be versioned, the version should be aligned with the EaseProbe binary version.
+Ease Probe supports multiple configuration files, just simply using `-f` indicate the directory of the configuration files.
```
Comment (proditis) [inline docs/Manual.md:1291]:
```suggestion
EaseProbe supports multiple configuration files, with the use of the command line flag `-f` which indicates the directory of the configuration files.
```

**Review by proditis — COMMENTED** [review]
:+1: minor edits

**Review by samanhappy — APPROVED** [review]

**Pull request merged**